<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sandblaasing_motta</class>
 <widget class="QWidget" name="sandblaasing_motta">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="side_sandblaasing_bedrifter">
      <widget class="QLabel" name="modus_0">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>671</width>
         <height>25</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Valgt modus: Motta | Annet | Sandblåsing</string>
       </property>
      </widget>
      <widget class="QPushButton" name="bilskadesenteret">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>240</y>
         <width>561</width>
         <height>221</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Bilskadesenteret Ringerike AS</string>
       </property>
      </widget>
      <widget class="QPushButton" name="nct">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>240</y>
         <width>561</width>
         <height>221</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Norwegian Coating Technologies (NCT)</string>
       </property>
      </widget>
      <widget class="QPushButton" name="tilbake_0_sandblaasing_bedrifter">
       <property name="geometry">
        <rect>
         <x>1110</x>
         <y>650</y>
         <width>111</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="logo_tepvr_0">
       <property name="geometry">
        <rect>
         <x>760</x>
         <y>0</y>
         <width>481</width>
         <height>94</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="side_sandblaasing_bilskadesenteret">
      <widget class="QPushButton" name="tilbake_1_sandblaasing_bilskadesenteret">
       <property name="geometry">
        <rect>
         <x>1110</x>
         <y>650</y>
         <width>111</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="modus_1_1">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>471</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Bedrift: Bilskadesenteret Ringerike AS</string>
       </property>
      </widget>
      <widget class="QLabel" name="modus_1">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>621</width>
         <height>25</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Valgt modus: Motta | Annet | Sandblåsing</string>
       </property>
      </widget>
      <widget class="QLabel" name="logo_tepvr_1">
       <property name="geometry">
        <rect>
         <x>760</x>
         <y>0</y>
         <width>481</width>
         <height>94</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="side_sandblaasing_nct">
      <widget class="QLabel" name="modus_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>621</width>
         <height>25</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Valgt modus: Motta | Annet | Sandblåsing</string>
       </property>
      </widget>
      <widget class="QLabel" name="modus_2_1">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>471</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Bedrift: Norwegian Coating Technologies (NCT)</string>
       </property>
      </widget>
      <widget class="QLabel" name="logo_tepvr_2">
       <property name="geometry">
        <rect>
         <x>760</x>
         <y>0</y>
         <width>481</width>
         <height>94</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="tilbake_2_sandblaasing_nct">
       <property name="geometry">
        <rect>
         <x>1110</x>
         <y>650</y>
         <width>111</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
